script.js

```diff
@@ -356,4 +356,47 @@ form.addEventListener('submit', (e) => {
     return false;
   }
   return form.submit();
+});
+
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you stick to the design as much as possible. the spacing, the emplacement of elements must be similar to the design in Figma.

`Hint: You may consider the flexbox to place your element on the page`
Your design :point_down: 
![Screenshot from 2023-01-06 16-06-26](https://user-images.githubusercontent.com/97100091/211028472-048b6cf4-4961-423f-b637-ed6e6ef5e72f.png)

The Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/211028947-f79827ac-1f00-4043-9cbd-ffa93c9cfe67.png)

Reply:
@christianbiring1 I am not getting your review clearly. Are you supposed to review the preserve data in the browser pull request or the contact form validation which has already been approved and I have noted the changes that I will make?

Reply:
@christianbiring1 What is your review for the preserve data in the browser pull request and why are all these change requests optional but you have blocked the project from being completed?

Reply:
- @chepkok3 I couldn't approve the project and some test cases for the previous milestone were not working correctly.
Apart from your project being approved by the previous reviewer maybe while implementing the new changes you broke the code and that's why those misfunctioning